xonsh/commands_cache.py

```diff
@@ -462,39 +467,51 @@ def default_threadable_predictors():
         "less": predict_help_ver,
         "ls": predict_true,
         "man": predict_help_ver,
+        "mc": predict_false,
         "more": predict_help_ver,
-        "mvim": predict_help_ver,
         "mutt": predict_help_ver,
+        "mvim": predict_help_ver,
         "nano": predict_help_ver,
+        "nmcli": predict_true,
         "nvim": predict_false,
         "ponysay": predict_help_ver,
         "psql": predict_false,
+        "push": predict_shell,
+        "pv": predict_false,
         "python": predict_shell,
         "python2": predict_shell,
         "python3": predict_shell,
-        "repo": predict_help_ver,
         "ranger": predict_help_ver,
+        "repo": predict_help_ver,
         "rview": predict_false,
         "rvim": predict_false,
+        "rwt": predict_shell,
         "scp": predict_false,
         "sh": predict_shell,
         "ssh": predict_false,
         "startx": predict_false,
         "sudo": predict_help_ver,
         "sudoedit": predict_help_ver,
+        "systemctl": predict_true,
         "tcsh": predict_shell,
         "telnet": predict_false,
-        "tput": predict_false,
         "top": predict_help_ver,
+        "tput": predict_false,
+        "udisksctl": predict_true,
+        "unzip": predict_true,
         "vi": predict_false,
         "view": predict_false,
         "vim": predict_false,
         "vimpager": predict_help_ver,
         "weechat": predict_help_ver,
+        "wget": predict_true,
         "xclip": predict_help_ver,
         "xo": predict_help_ver,
-        "xonsh": predict_shell,
         "xon.sh": predict_shell,
-        "zsh": predict_shell,
+        "xonsh": predict_shell,
+        "yes": predict_false,
+        "zip": predict_true,
+        "zipinfo": predict_true,
+        "zsh": predict_shell
```

Comment:
`black` wants a trailing comma at the end of this line
```suggestion
        "zsh": predict_shell,
```